addons/web/static/src/search/control_panel/control_panel.scss

```diff
@@ -202,12 +202,6 @@
                 cursor: pointer;
             }
         }
-
-        > .o_cp_switch_buttons {
-            .dropdown-menu {
-                right: 0;
```

Comment:
this is the actual fix in this commit, but fixing it shed a light to the usePosition hook...